dali/python/nvidia/dali/ops.py

```diff
@@ -711,15 +724,12 @@ def _repack_list(self, sets, fn):
             return output_list
 
         def _check_schema_num_inputs(self, inputs):
-            if (len(inputs) > self._schema.MaxNumInput() or
-                    len(inputs) < self._schema.MinNumInput()):
+            if (len(inputs) > self._schema.MaxNumInput()
+                    or len(inputs) < self._schema.MinNumInput()):
                 raise ValueError(
-                    ("Operator {} expects from {} to " +
-                    "{} inputs, but received {}.")
-                    .format(type(self).__name__,
-                            self._schema.MinNumInput(),
-                            self._schema.MaxNumInput(),
-                            len(inputs)))
+                    ("Operator {} expects from {} to " + "{} inputs, but received {}.").format(
+                        type(self).__name__, self._schema.MinNumInput(), self._schema.MaxNumInput(),
+                        len(inputs)))
```

Comment:
The original code was much more readable (even if not adhering to the guidelines).
```suggestion
                    f"Operator {type(self).__name__} expects "
                    f"from {self._schema.MinNumInput()} to {self._schema.MaxNumInput()} inputs, "
                    f"but received {len(inputs)}.")
```

Reply:
Done